images/openwisp_api/urls.py

```diff
@@ -27,7 +27,7 @@
             'api/v1/',
             include('openwisp_firmware_upgrader.api.urls', namespace='firmware'),
         ),
-        path('', include((fw_private_storage_urls, 'firmware'), namespace='firmware')),
+        path('', include((fw_private_storage_urls, 'firmware'), namespace='firmware-private')),
```

Comment:
doesn't look consistent with what we have in ansible-openwisp2:

https://github.com/openwisp/ansible-openwisp2/blob/master/templates/openwisp2/urls.py#L26-L32